homing_missiles.py

```diff
@@ -108,6 +125,27 @@ def draw_entities(self, screen):
                                           main_character.size_ * self.DRAWING_SCALE_)
         pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
         self.draw_friendly_projectiles(screen)
+        self.draw_enemy_projectiles(screen)
```

Comment:
Can you change the order of entity drawing so that projectiles are first, enemies in the middle, main character last?